package.json

```diff
@@ -15,7 +15,7 @@
     "lint": "eslint \"**/*.{js,ts}\"",
     "nuke": "rm -rf node_modules && npm install && lerna exec 'rm -rf node_modules' && lerna bootstrap && npm run clean",
     "upgrade": "rm -rf node_modules package-lock.json && npm install && lerna exec 'rm -rf node_modules package-lock.json' && lerna bootstrap && npm run clean",
-    "test": "lerna run test --ignore lit --ignore lit-html --ignore lit-element --ignore @lit/reactive-element --ignore @lit-labs/motion --concurrency 1 --stream",
+    "test": "cd packages/tests && npm run test",
```

Comment:
IIUC, it seems like this might cause our `tests-local` job to stop including any of our Node tests. For example the `ssr`, `localize-tools`, `ts-transformers` packages all have Node tests which aren't handled by the consolidated `tests/` package.

https://github.com/lit/lit/blob/3e181bcb3d969775eda799fd6fcae1ead843225b/.github/workflows/tests.yml#L74

One potential short term fix for this PR could be to add some more `--ignore` statements to the `lerna` command, instead of dropping the lerna command (so, an additional `--ignore` for each of the new packages that have been added to the consolidated `tests` config, minus ssr [see other comment]).

Reply:
I'd like to consolidate `--ignore` statements into a single `--ignore` similar to [this setup](https://github.com/lit/lit/blob/38bdf5cb47a1d76bb7a5cb2e5afae5dc43ae282e/package.json#L19)

Reply:
take advantage of pattern matching, looks more readable too imho
```
"lerna run test:dev --ignore '{lit,tests,@lit-labs/motion}' --concurrency 1 --stream ",
```

Reply:
Cool. Yeah that is more readable, I agree. (TIL Lerna supports glob-style braces in its flags).

Reply:
That needs to be `"lerna run test --ignore ..."`, yeah?

Reply:
added packages tested in `tests/` to the lerna ignore line!
Gonna resolve this and request a new review :)